### PR TITLE
Create 15_how-do-i-upload-a-writing

### PR DESCRIPTION
**Pull request by DarklingMuse**
From updates in FAQ doc

### CODE REVIEW

File: 03_pictures-videos-and-writings/15_how-do-i-upload-a-writing [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#How do I upload a Writing to my profile?
```
Comment (sylee-) [inline 03_pictures-videos-and-writings/15_how-do-i-upload-a-writing:1]: Add a space between # and How

File: 03_pictures-videos-and-writings/15_how-do-i-upload-a-writing [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+There are three ways to upload Writings to your profile on FetLife. 
+
+Select the hamburger menu (icon with three horizontal lines) on the upper right corner of the page.
```
Comment (sylee-) [inline 03_pictures-videos-and-writings/15_how-do-i-upload-a-writing:5]: Select the hamburger menu (icon with three horizontal lines) on the upper right corner of the page.  Select "Write Post" from the four boxes available at the top of the drop-down list.

**Comment by DarklingMuse** [comment]
Do we just need to link to https://fetlife.com/posts/new here and talk through the options instead of explaining how to get there? Since we're not trying to teach them how to use the site?

**Comment by nielsenramon** [comment]
Closing for now as it's not written in our template

**Pull request closed without merging**